[Bug] g.gui: AttributeError: 'ModuleNode' object has no attribute '_children'

**Describe the bug**
Starting `g.gui` in current master (I used `make distclean`), it crashes with:

```
GRASS 7.9.dev (nc_spm_08_grass7):~ > g.gui
Launching <wxpython> GUI in the background, please wait...
Traceback (most recent call last):
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/wxgui.py", line 105, in OnInit
    mainframe = GMFrame(parent=None, id=wx.ID_ANY,
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/lmgr/frame.py", line 143, in __init__
    self.notebook = self._createNoteBook()
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/lmgr/frame.py", line 353, in _createNoteBook
    menuModel=self._moduleTreeBuilder.GetModel(),
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/menutree.py", line 162, in GetModel
    removeSeparators(model)
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/menutree.py", line 189, in removeSeparators
    removeSeparators(model, child)
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/menutree.py", line 189, in removeSeparators
    removeSeparators(model, child)
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/menutree.py", line 189, in removeSeparators
    removeSeparators(model, child)
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/menutree.py", line 191, in removeSeparators
    model.RemoveNode(node)
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/treemodel.py", line 137, in RemoveNode
    node.parent.children.remove(node)
  File "/home/mneteler/software/grass_master/dist.x86_64-pc-linux-gnu/gui/wxpython/core/treemodel.py", line 175, in children
    return self._children
AttributeError: 'ModuleNode' object has no attribute '_children'
OnInit returned false, exiting...
Error in atexit._run_exitfuncs:
wx._core.wxAssertionError: C++ assertion "GetEventHandler() == this" failed at ../src/common/wincmn.cpp(477) in ~wxWindowBase(): any pushed event handlers must have been removed
```

**System description:**

Fedora 32

```
g.version -rge
version=7.9.dev
date=2020
revision=a3888a00d
build_date=2020-07-12
build_platform=x86_64-pc-linux-gnu
build_off_t_size=8
libgis_revision=229dd3ea7
libgis_date=2020-06-03T16:03:52+00:00
proj=6.3.2
gdal=3.0.4
geos=3.8.1
sqlite=3.32.2

python --version
Python 3.8.3

rpm -qa | grep wxpython
python3-wxpython4-4.0.7-5.fc32.x86_64
```

--- a/gui/wxpython/core/treemodel.py
+++ b/gui/wxpython/core/treemodel.py
@@ -80,7 +80,9 @@ class TreeModel(object):
         # useful for debugging deleting nodes
         # weakref.finalize(node, print, "Deleted node {}".format(label))
         parent.children.append(node)
-        node.parent = weakref.proxy(parent)
+        # weakref doesn't work out of the box when deepcopying this class
+        # node.parent = weakref.proxy(parent)
+        node.parent = parent
         return node
 
     def SearchNodes(self, parent=None, **kwargs):

